### PR TITLE
feature: add size for humanize byte size

### PR DESCRIPTION
**Pull request by jsvisa**
- [x] Rebased/mergable
- [x] Tests pass
- [x] CHANGELOG.md updated

### CODE REVIEW

File: tick/stateful/functions.go [hunk 1]
```diff
@@ -598,3 +602,27 @@ func (*year) Call(args ...interface{}) (v interface{}, err error) {
...
+func (*size) Call(args ...interface{}) (v interface{}, err error) {
+	if len(args) != 1 {
+		return 0, errors.New("size expects exactly one argument")
+	}
+	switch a := args[0].(type) {
+	case float64:
+		v = humanize.Bytes(uint64(int64(a)))
+	case int64:
+		v = humanize.Bytes(uint64(a))
+	case uint64:
```
Comment (nathanielc) [inline tick/stateful/functions.go]: This case isn't necessary as all ints will be int64s.

File: tick/stateful/functions.go [hunk 2]
```diff
@@ -598,3 +602,27 @@ func (*year) Call(args ...interface{}) (v interface{}, err error) {
...
+
+}
+
+func (*size) Call(args ...interface{}) (v interface{}, err error) {
+	if len(args) != 1 {
+		return 0, errors.New("size expects exactly one argument")
+	}
+	switch a := args[0].(type) {
+	case float64:
+		v = humanize.Bytes(uint64(int64(a)))
```
Comment (nathanielc) [inline tick/stateful/functions.go]: This can simplify to:

```
v = humanize.Bytes(uint64(a))
```

File: tick/stateful/functions.go [hunk 3]
```diff
@@ -83,6 +84,9 @@ func init() {
 	statelessFuncs["day"] = &day{}
 	statelessFuncs["month"] = &month{}
 	statelessFuncs["year"] = &year{}
+
+	// Humanize functions
+	statelessFuncs["size"] = &size{}
```
Comment (nathanielc) [inline tick/stateful/functions.go]: Should the function name be something like `humanBytes` or something along those lines?

Comment (jsvisa) [inline tick/stateful/functions.go]: Sorry for the delay,  I'll adjust those change right now.

**Comment by jsvisa** [comment]
@nathanielc Apologies for the delay. I'm just updated as what you mentioned.

**Comment by nathanielc** [comment]
@jsvisa Thanks!

**Comment by BhuvanaBal** [comment]
may i know from which kapacitor version the function humanize is supported. could you please let me know
Additional could you please let me know how can I view  the kapacitor documentation for 1.5.6 . If someone has a backup of the documentation could you please share it

**Pull request closed without merging**